package.json

```diff
@@ -1,27 +1,27 @@
 {
     "name": "ufds",
-    "version": "7.3.1",
+    "version": "7.3.2",
     "repository": {
         "type": "git",
-        "url": "git@github.com:joyent/sdc-ufds.git"
+        "url": "git@github.com:TritonDataCenter/sdc-ufds.git"
     },
     "private": true,
     "main": "./main.js",
     "engines": {
         "node": ">=0.10"
     },
     "dependencies": {
-        "aperture": "git+https://github.com/joyent/node-aperture.git#516f54924f9b3c997cca674e92937250801bd037",
-        "aperture-config": "git+https://github.com/joyent/aperture-config.git#448811fa1696cd6c63d8d3391187c084b39b2e7b",
+        "aperture": "git+https://github.com/TritonDataCenter/node-aperture.git#516f54924f9b3c997cca674e92937250801bd037",
+        "aperture-config": "git+https://github.com/TritonDataCenter/aperture-config.git#752b0a27192ae091e382b55f5bc9ec5a806477be",
         "asn1": "^0.2.4",
         "assert-plus": "0.1.2",
         "async": "0.1.22",
         "backoff": "2.4.1",
-        "bcrypt": "0.7.8",
+        "bcrypt": "1.0.3",
         "bunyan": "0.22.1",
         "crc": "0.2.0",
-        "ldapjs": "git+https://github.com/mcavage/node-ldapjs.git#0509dd44c0eb6dad7235df7838bdb05ea31cde14",
-        "moray": "git+https://github.com/joyent/node-moray.git#debd4617e6810241f51d9256470e7383ab4c2ffd",
+        "ldapjs": "git+https://github.com/ldapjs/node-ldapjs.git#0509dd44c0eb6dad7235df7838bdb05ea31cde14",
```

Comment:
Should we pull this from NPM?

[0509dd44c0eb6dad7235df7838bdb05ea31cde14](https://github.com/ldapjs/node-ldapjs/blob/0509dd44c0eb6dad7235df7838bdb05ea31cde14/package.json) looks to be at version: 0.7.1